object/attribute.go

```diff
@@ -30,8 +30,8 @@ const (
 type Attribute struct{ k, v string }
 
 // NewAttribute creates and initializes new [Attribute].
-func NewAttribute(key, value string) *Attribute {
-	return &Attribute{key, value}
+func NewAttribute(key, value string) Attribute {
```

Comment:
In general `New*` return pointers. But this struct is quite small and primarily it's used for `SetAttributes()`, so I'm OK to have it non-pointer here.